cypress/integration/pages/testsForAllPages.js

```diff
@@ -63,11 +63,118 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
           );
         });
 
-        it('should have the correct shared social media metadata', () => {
-          cy.checkSharedSocialmediaMetadata({
-            fbAdmins: '100004154058350',
-            appID: '1609039196070050',
+        it('should have the correct shared metadata', () => {
+          cy.get('head').within(() => {
+            cy.get('meta[name="fb:admins"]').should(
```

Comment:
I think this can be simplified by adding all metaTags to a list then iterating over the list.

`const metaTags = [
              {
                name: 'fb:admins',
                value: '100004154058350',
              },
            ];

            metaTags.forEach(({ name, value }) => {
              cy.get(`meta[name=${name}]`).should(
                'have.attr',
                'content',
                value,
              );
            });`

Reply:
Thanks for the suggestion. I did this locally and because of prettifier with eslint running with `metaTags`, it made the have the same number of lines of code. So I decided to leave the repetition since it reads more clearly not having that level of abstraction.